src/store/getters/resource-getters.ts

```diff
@@ -3,51 +3,67 @@ import {
   FilingDataIF,
   HelpSectionIF,
   IncorporationAgreementTypeIF,
-  ResourceIF,
+  StateIF,
   StepIF
 } from '@/interfaces'
 
-/** The company rules and values based on entity type. */
-export const getCompanyResources = (state: any): ResourceIF => {
-  return state.resourceModel
+//
+// The getters in this file return values from the current resource
+// model -- in other words, for the currently-selected entity type.
+//
+
+/** The company title. */
+export const getCompanyTitle = (state: StateIF): string => {
+  return state.resourceModel.title
+}
+
+/** The company description. */
+export const getCompanyDescription = (state: StateIF): string => {
+  return state.resourceModel.description
 }
 
-/** The company rules and values based on entity type. */
-export const getDisplayName = (state: any): string => {
+/** The company display name. */
+export const getCompanyDisplayName = (state: StateIF): string => {
   return state.resourceModel.displayName
 }
 
 /** The completing party statement resources. */
-export const getCompletingPartyStatement = (state: any): CertifyStatementIF => {
+export const getCompletingPartyStatement = (state: StateIF): CertifyStatementIF => {
   return state.resourceModel.reviewAndConfirm.completingPartyStatement
 }
 
 /** The incorporation agreement help info. */
-export const getIncorporationAgreementHelp = (state: any): Array<HelpSectionIF> => {
+export const getIncorporationAgreementHelp = (state: StateIF): Array<HelpSectionIF> => {
   return state.resourceModel.incorporationAgreement.helpSection
 }
 
 /** The incorporation agreement sample article. */
-export const getSampleArticle = (state: any): string => {
+export const getSampleArticle = (state: StateIF): string => {
   return state.resourceModel.incorporationAgreement.article
 }
 
 /** The incorporation agreement options. */
-export const getIncorporationAgreementDocuments = (state: any): Array<IncorporationAgreementTypeIF> => {
+export const getIncorporationAgreementDocuments = (state: StateIF): Array<IncorporationAgreementTypeIF> => {
   return state.resourceModel.incorporationAgreement.documents
 }
 
 /** The minimum directors required. */
-export const getMinimumDirectorCount = (state: any): number => {
+export const getMinimumDirectorCount = (state: StateIF): number => {
   return state.resourceModel.directors.countMinimum
 }
 
-/** Returns the array of steps. */
-export const getSteps = (state: any): Array<StepIF> => {
+/** The array of steps. */
+export const getSteps = (state: StateIF): Array<StepIF> => {
   return state.resourceModel.steps
 }
 
-/** Returns the resource filing data. */
-export const getFilingData = (state: any): FilingDataIF => {
+/** The maximum number of steps. */
+export const getMaxStep = (state: StateIF): number => {
```

Comment:
I moved this here from the other getters file since it operates on resource data (not state data).